xclim/sdba/measures.py

```diff
@@ -442,3 +442,68 @@ def _scorr(
 scorr = StatisticalPropertyMeasure(
     identifier="Scorr", aspect="spatial", compute=_scorr, allowed_groups=["group"]
 )
+
+
+def _taylordiagram(
+    sim: xr.DataArray,
+    ref: xr.DataArray,
+    dim: str = "time",
+    group: str | Grouper = "time",
+) -> xr.DataArray:
+    """Taylor diagram.
+
+    Compute the respective standard deviations of a simulation and a reference array, as well as the Pearson
+    correlation coefficient between both, all necessary parameters to plot points on a Taylor diagram.
+
+    Parameters
+    ----------
+    sim : xr.DataArray
+        data from the simulation (a time-series for each grid-point)
+    ref : xr.DataArray
+        data from the reference (observations) (a time-series for each grid-point)
+    dim : str
+        Dimension across which the correlation and standard deviation should be computed.
+    group : str
+        Compute the property and measure for each temporal groups individually.
+        Currently not implemented.
+
+
+    Returns
+    -------
+    xr.DataArray, [measures]
+        DataArray containing the correlation coefficient and standard deviations as coordinates.
```

Comment:
In this class, this line of the docstring becomes the `long_name`. Not necessarily an issue, but just wanted to let you know in case you need it for spirograph.